public/chirpy.js

```diff
@@ -13,6 +13,7 @@ function start () {
             console.error(error, status);
         } else {
             console.log(agent);
```

Comment:
Not required. The other agent prints out some similar info so I thought I'd keep it about in this version too.